Tests/SMTPKittenTests/SMTPKittenTests.swift

```diff
@@ -1,105 +1 @@
-import XCTest
```

Comment:
Given there are no tests, it is a lot harder to review as I've had to work out the API